vnext/Microsoft.ReactNative/IReactDispatcher.h

```diff
@@ -6,9 +6,22 @@
 #include <dispatchQueue/dispatchQueue.h>
 #include <winrt/Microsoft.ReactNative.h>
 
+namespace Mso::React {
+
+struct __declspec(uuid("a8d9db25-3d16-4476-ae65-682fcee1260c")) ISimpleDispatch : ::IUnknown {
```

Comment:
>ISimpleDispatch [](http://example.com/codeflow?start=64&length=15)

It would be nice to choose another name. Though I only have IDispatchQueue2 coming to mind. I hope you do not mind switching to another name should find it.

Reply:
Renamed to IDispatchQueue2 for now.  We can revisit modifying the DQ interface in later changes when not backporting everything.